package.json

```
@@ -40,7 +40,7 @@
     "@ngx-translate/core": "^15.0.0",
     "@nx/angular": "^20.4.2",
     "@vendure/ngx-translate-extract": "^9.0.3",
-    "geonetwork-ui": "2.4.1",
+    "geonetwork-ui": "^2.4.2-dev.e01cfe9b9",
```

Please keep exact versions here, as otherwise npm can get mixed up with versions containing a hash.
```suggestion
    "geonetwork-ui": "2.4.2-dev.e01cfe9b9",
```